### PR TITLE
copy edits on homepage

### PR DESCRIPTION
**Pull request by saddington**
i couldn't help myself... the first is what we call a typesetting issue with a "widowed" or "orphaned" word (in this case "love") that's all by itself on its own line. It's just better to not have one word alone so adding the word "use" makes this look better.

the second is the use of "Internet" rather than "Information" which is typically used in conjunction with "age" (and it should be capitalized). "Information Age" is what is more commonly used.

next... I've made some small edits for flow and readability, oxford commas, consistent capitalization and word counts... etc... etc... ... these are obviously up for debate.

### CODE REVIEW

**Comment by chrislloyd** [comment]
Hey @saddington! Love the changes. However, I feel a little uncomfortable with the word "meritocracy". I feel your general point there but there [lots of](http://readwrite.com/2014/01/24/github-meritocracy-rug) [other narratives](http://geekfeminism.org/2009/11/29/questioning-the-merit-of-meritocracy/) associated with it so it's probably best to try and re-phrase the point. Maybe the rest of the paragraph could be re-written?

**Comment by chrislloyd** [comment]
(Also, welcome & hi!)

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -196,7 +196,7 @@
       <div class="main">
         <h2>
 			Where we band together to build
-			products people love.
+			products people use & love.
```
Comment (pletcher) [inline app/views/pages/about.html.erb]: I'm not sure about the ampersand vs. "and" -- @awwstn, what's our copy style guide say?

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -392,11 +392,11 @@
 
           	<div class="sm-col sm-col-6 px2">
           		<h4>Global Diversity</h4>
-          		Everyone is invited to make an impact, have influence, and be able to share their best work. Ideas, vision and execution are be driven by the community.
+          		Everyone is invited to make an impact, leverage their influence, and share their best work. Ideas, vision, and execution are being entirely driven by (and for) the community.
           	</div>
 
           	<div class="sm-col sm-col-6 px2">
-          		<h4>Explicitness</h4>
+          		<h4>Real Meritocracy</h4>
```
Comment (pletcher) [inline app/views/pages/about.html.erb]: Seconding @chrislloyd here.

**Comment by saddington** [comment]
love it!

i had actually meant to change "&" to "and"... i saw that you folks don't use that much. whoops.

and... spot-on. i struggled and sat on the "meritocracy" for a long time... i wasn't completely comfortable using it either but I threw it in there and i'm glad it's a conversation point. i'm not sure what it should be, to be honest... you fine folks would know better. sweet!

**Comment by chrislloyd** [comment]
hey @saddington if you revert the "meritocracy" change happy to give a :+1:

**Comment by saddington** [comment]
whoot.

**Pull request merged**